taskvine/src/manager/vine_perf_log.c

```diff
@@ -52,7 +53,7 @@ void vine_perf_log_write_update( struct vine_manager *q, int force )
 	struct vine_stats s;
 
 	timestamp_t now = timestamp_get();
-	if(!force && (now - q->time_last_log_stats < ONE_SECOND)) {
+	if(!force && (now - q->time_last_log_stats < (ONE_SECOND*VINE_PERF_LOG_INTERVAL))) {
```

Comment:
Make this a variable `q->perf_log_interval` modifiable by `vine_tune` instead of a macro.

I am certain that we will run into issues at scale where we will want to increase/decrease this interval periodically.